apps/gallery/style/VideoPlayer.css

```diff
@@ -7,16 +7,6 @@
 }
 
 /* video player controls */
-.videoPlayerControls {
-  position: absolute;
-  left: 0;
-  right: 0;
-  top: 0;
-  bottom: 0;
-  margin: 0;
-  padding: 0;
-}
-
```

Comment:
Since this is being removed entirely, a cleaner patch would remove the element entirely from video_player.js as well. But that would require more changes to this basically obsolete code, and I guess it is better to keep the patch as simple as possible.

It took me a while (and some test code) to convince myself that the VideoPlayer show() and hide() methods would continue to work to hide the play button and footer even though they were no longer visually within this controls container element.